### DISCUSSION TITLE
Request for Support of Armenian Language in MeiliSearch

### DISCUSSION THREAD

**Discussion by NarHakobyan** [meilisearch/product#662] [Feedback & Feature Proposal]
Hi MeiliSearch Team,

We're using MeiliSearch for a project with a significant Armenian user base, and we've noticed there isn't explicit support for the Armenian language in the current language configuration settings.
Could you please inform us if there are plans to support Armenian in the future?

**Comment by irevoire** [comment]
Hey,
Isn't Armenian a space-separated language?
If that's the case, then it should work pretty well with meilisearch

**Reply by NarHakobyan** [reply]
Yeah, but search is case sensitive :/, for example Երևան and երևան gives different results.

Եե are upper and lower case letters.

**Comment by curquiza** [comment]
Hey @NarHakobyan I move this discussion to the product repo where we handle the product request, thank you for your feedback
